src/familiar/itemFamiliar.ts

```diff
@@ -0,0 +1,66 @@
+import { Familiar, myFamiliar, runChoice, useFamiliar, visitUrl } from "kolmafia";
+import { $familiar, $item, findFairyMultiplier, get, have, set } from "libram";
+import { menu } from "./freeFightFamiliar";
+
+let bestNonCheerleaderFairy: Familiar;
+
+export function bestFairy(): Familiar {
+  if (have($familiar`Trick-or-Treating Tot`) && have($item`li'l ninja costume`)) {
+    return $familiar`Trick-or-Treating Tot`;
+  }
+
+  if (
+    have($familiar`Steam-Powered Cheerleader`) &&
+    findFairyMultiplier($familiar`Steam-Powered Cheerleader`) > 1.25
+  ) {
+    return $familiar`Steam-Powered Cheerleader`;
+  }
+
+  if (!bestNonCheerleaderFairy) {
+    const viableFairies = Familiar.all()
+      .filter(
+        (f) => have(f) && findFairyMultiplier(f) && f !== $familiar`Steam-Powered Cheerleader`
+      )
+      .sort((a, b) => findFairyMultiplier(b) - findFairyMultiplier(a));
+
+    const highestFairyMult = findFairyMultiplier(viableFairies[0]);
```

Comment:
Will this crash if a Guy does not have any fairies at all?

Reply:
yes. I'm very fine with that